src/platform/qt/CMakeLists.txt

```diff
@@ -481,3 +480,23 @@ if(DISTBUILD AND NOT APPLE)
 		add_custom_command(TARGET ${BINARY_NAME}-qt POST_BUILD COMMAND "${STRIP}" "$<TARGET_FILE:${BINARY_NAME}-qt>")
 	endif()
 endif()
+
+if(BUILD_SUITE)
+	enable_testing()
+	find_package(${QT}Test)
+	if(${QT}Test_FOUND)
+		set(TEST_SRC
+			test/main.cpp
```

Comment:
Nit: Alphabetical order

Reply:
How does sort order apply to files vs folders? Files first, folders first, sort strictly alphabetically?

Reply:
Strictly alphabetically